tuf/api/serialization/__init__.py

```diff
@@ -15,6 +15,11 @@
 
 """
 import abc
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    # pylint: disable=cyclic-import
+    from tuf.api.metadata import Metadata, Signed
```

Comment:
So mypy recognizes that the quoted strings "Metadata" and "Signed" are the classes in this import?
Cool trick!

Reply:
Yes, AFAIK for mypy the type-names-as-string are just as valid.